OPHD/States/MapViewStateTurn.cpp

```diff
@@ -372,7 +372,10 @@ void MapViewState::countFood()
 {
 	mFood = 0;
 
-	const auto& structures = NAS2D::Utility<StructureManager>::get().structureList(Structure::StructureClass::FoodProduction);
+	auto structures = NAS2D::Utility<StructureManager>::get().structureList(Structure::StructureClass::FoodProduction);
+	auto& command = NAS2D::Utility<StructureManager>::get().structureList(Structure::StructureClass::Command);
```

Comment:
Nothing wrong here, just wanted to comment since it wasn't obvious and I had to look it up. The `auto` keyword in `auto& command = ...` will deduce `const`. Technically the `const` in the removed line `const auto& structures = ...` could have been removed. If the reference was not there, the situation would be different as a copy would be made, and a copy is not required to retain the const-ness of the original.

Reply:
This is a good point. It's easy enough for me to know because Visual Studio does show how it evaluates so I know it's const. I may change that later for the sake of being obvious.